tests/src/test/scala/com/nvidia/spark/rapids/DecimalBinaryOpSuite.scala

```diff
@@ -119,4 +120,10 @@ class DecimalBinaryOpSuite extends GpuExpressionTestSuite {
     checkEvaluateGpuUnaryExpression(GpuLessThanOrEqual(lit, leftExpr),
       schema.head.dataType, DataTypes.BooleanType, expectedFunSV, schema)
   }
+
+  testSparkResultsAreEqual("SPARK-24957: average with decimal followed by " +
```

Comment:
nit: Instead of using Spark issue or in addition to it we should also use our issue # for reference here.